test/test-exec-deserialization.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/python3
+
+#
+#  Copyright 2017 Michal Sekletar <msekleta@redhat.com>
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+#
+#  systemd is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with python-systemd; If not, see <http://www.gnu.org/licenses/>.
+
```

Comment:
I didn't notice this comment. Sure I can add similar comment to my test. Even though I think it is kinda obvious that if you don't do ``` make && sudo make install && sudo init u``` then of course you will be testing your old systemd version and not the git version.